src/common/components/vanilla/table-interactive/body.js

```diff
@@ -0,0 +1,14 @@
+import React, { PropTypes } from 'react';
+import styles from './table.css';
+
+export default function Body(props) {
+  return (
+    <main className={ styles.body }>
```

Comment:
GH is swallowing comments on outdated diffs, but we discussed it here:
https://github.com/canonical-ols/build.snapcraft.io/pull/137#discussion_r101288714

Styling table elements may get tricky, especially that we want them to expand to allow showing additional info and register name form.